content/en/blog/sign-verify-plugins-rules/index.md

```diff
@@ -0,0 +1,180 @@
+
+#### Safeguards against registry tampering, secure by default
+
+Making use of container registries to distribute artifacts is convenient, and allows for easy replication of the data across multiple regions or providers. But even if one of those registries is at [risk of being compromised](https://www.bleepingcomputer.com/news/security/amazon-ecr-public-gallery-flaw-could-have-wiped-or-poisoned-any-image/) , clients will be protected from potentially malicious applications. If **an artifact is replaced in the registry** without going through the appropriate approval and open source release pipeline, **clients will identify that the signature does not match and automatically refuse to install** any compromised package. This is only possible because cosign signature information is distributed in a file that is separate from the artifact itself. With falcoctl 0.6.1 this feature is on by default and requires no extra action from users.
+
+#### Independent signature verification
+
+It is now possible for **all Falco rules and plugins users to independently verify the authenticity** of every artifact they download. We implemented this feature staying true to the open nature of Falco, so by design, you do not need any Falco-specific tool (like `falcoctl`) to verify your artifact and you can do it manually with `cosign` to implement even tighter controls.
+
+
+#### Attack surface
+
+As security practitioners, we should always analyze the attack surface of the security schemes we design and update our threat model accordingly. In this case, we can identify that:
+
+* If the source repositories are compromised, so will the packages. Of course, the open-source repositories act as the single source of truth for our plugins and rules. If an attacker manages to merge malicious code and release a malicious plugin they will be able to compromise users once they update or install a new version. Release pipelines are the only component that can generate a proper signature for the artifacts.
+* If the index file is compromised, so will the packages. The index file maps artifact names (e.g. `cloudtrail`) to their location in the container registry (e.g. `ghcr.io/falcosecurity/plugins/plugin/cloudtrail`). If that entry is modified to point to an attacker-controlled registry, possibly alongside an attacker-controlled signature, clients will download a potentially malicious artifact instead of the official one.
+* At this point in time, the entire infrastructure is hosted by GitHub, making it a potential single point of failure and an entity that the Falco project completely trusts. However, even if parts of the infrastructure (container registry, CDN for index distribution, ...) were not, GitHub release pipelines would still have permissions or access tokens to operate those locations.
+
```

Comment:
Also, we could conclude the blog with something like this:

```suggestion
## Le'ts in touch

Given the above considerations, this is a work in progress toward a more secure supply chain for Falco and its artifacts.

We'll keep you up to date with the next steps and remember, if you want to be involved you're more than welcome, don't hesitate to reach out to the Falco Software Supply Chain Security Working Group in the [#falco-sscs-wg](https://cloud-native.slack.com/archives/C04ET5YR014) Slack channel in the CNCF workspace.
```

Reply:
Done!